R/overrideGraphics.R

```diff
@@ -14,11 +14,16 @@ overrideGraphics <- function(name, object, ...) {
         params <- append(list(object), params)
     }
     
-    
-    defFun <- getFromNamespace(ifelse(existsFunction(paste0(name,".default")), paste0(name,".default"), name), 'graphics')
+    base.package <- ifelse(length(grep(pattern="error_bar", x=name)) > 0, "gsplot", "graphics")
+    defFun <- getFromNamespace(ifelse(existsFunction(paste0(name,".default")), paste0(name,".default"), name), base.package)
```

Comment:
hmm...yeah, in my head I made this more generalized...apparently not in real life